draft-ietf-core-yang-cbor-latest.md

```diff
@@ -1608,10 +1618,12 @@ CBOR encoding:
 
 # Content-Types {#content-type}
 
-The following Content-Type is defined:
+Based on the media-type `application/yang-data+cbor`, the following
+Content-Types are defined:
```

Comment:
Should this section say something about future additions to the set of possible values of the `id` parameter?

Reply:
We don't know anything about that future.
What can we say at this point?
Any additional id value would be an update as per section 5.5 of RFC 6383.
(See also the internally inconsistent section 4.3.)

Reply:
> What can we say at this point?

Are future additions possible at all? And, if yes, what's the ABNF for potential parameter values?

That's the two things that come to my mind.

Reply:
OK, had a crystal ball handy.